src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckPolicyCollector.java

```diff
@@ -44,12 +47,19 @@ public BlackDuckPolicyCollector(final JsonExtractor jsonExtractor, final List<Me
         super(jsonExtractor, messageContentProcessorList, contentTypes);
     }
 
-    protected void addApplicableItems(final List<CategoryItem> categoryItems, final Long notificationId, final LinkableItem policyItem, final String policyUrl, final ItemOperation operation, final SortedSet<LinkableItem> applicableItems) {
-        final List<String> keyItems = Stream.concat(applicableItems.stream().map(LinkableItem::getValue), Stream.of(policyUrl)).collect(Collectors.toList());
+    protected void addApplicableItems(final List<CategoryItem> categoryItems, final Long notificationId, final Set<LinkableItem> policyItems, final ItemOperation operation, final Set<LinkableItem> applicableItems) {
+        final List<String> keyItems = Stream.concat(
+            applicableItems.stream().map(LinkableItem::getValue),
```

Comment:
Is the "value" of the LinkableItem a url? If so, we name the collection in a way that makes it obvious; if not, why are we collecting a group of urls and "unknown" items?